applications/mlflow/DEVELOPMENT.md

```diff
@@ -0,0 +1,215 @@
+# MLflow Development Guide
+
+This document provides information about developing, testing, and releasing the MLflow application using the included Taskfile.
+
+## Development Workflow
+
+The MLflow application includes a Taskfile.yml that provides tasks for developing, testing, and publishing the application.
+
+### Prerequisites
+
+- [Task](https://taskfile.dev/#/installation) command line tool
+- Kubernetes cluster configured in your current context
+- kubectl, helm, and python3 installed
+
+### Local Development
+
+Follow this workflow for development:
+
+1. Add required Helm repositories and update dependencies:
+   ```bash
+   task add:repos:helm
+   task update:deps:helm
```

Comment:
Interesting is this the actual command? So rather than helm-add-repos or helm-update-deps the command had ":" in it? Is that because of how the tasks are organized/imported or just an intentional naming decision?

I don't know that I have any particular opinion on the right way but this at least looked different enough to me to stand out.